libkbfs/folder_branch_ops.go

```diff
@@ -4515,6 +4520,27 @@ func (fbo *folderBranchOps) finalizeResolutionLocked(ctx context.Context,
 	default:
 	}
 
+	if jServer, err := GetJournalServer(fbo.config); err == nil {
+		// Disable journal writes after flushing all the resolution
+		// block writes -- resolutions must go straight through to the
+		// server or else the journal will get confused.
+		if err = jServer.Wait(ctx, fbo.id()); err != nil {
+			return err
+		}
+		wasEnabled, err := jServer.Disable(ctx, fbo.id())
```

Comment:
I think so...you can put it behind an accessor if you want (or leave that for later, if it gets more complicated)